e2e/nomostest/registryproviders/helm.go

```diff
@@ -44,37 +50,69 @@ func HelmChartVersion(version string) func(options *helmOptions) {
 	}
 }
 
+// HelmChartDir builds the chart with the specified chart input directory.
+func HelmChartDir(dir string) func(options *helmOptions) {
```

Comment:
I think I'd prefer this interface just take the chart name rather than an arbitrary dir path. If this function assumes that all charts are in the `testdata/helm-charts` directory I think that will nudge future authors to keep things consistent.

Reply:
Make sense. Added a comment for the parent directory of the source chart and source package.